packages/block-library/src/embed/wp-embed-preview.js

```diff
@@ -13,7 +13,7 @@ const attributeMap = {
 	marginwidth: 'marginWidth',
 };
 
-export default function WpEmbedPreview( { html } ) {
+export default function WpEmbedPreview( { html, className, style } ) {
```

Comment:
As noted in my earlier comment, these new props might be better combined into a single `wrapperProps` prop or something.